lib/pbench/test/unit/server/test_datasets_list.py

```diff
@@ -48,7 +49,12 @@ def query_api(
             """
             headers = None
             if username:
-                token = self.token(client, server_config, username)
+                token = generate_token(
+                    username=username,
+                    pbench_client_roles=["ADMIN"]
+                    if username == admin_username
+                    else None,
+                )
```

Comment:
We have the `token` method down below that does this in a more readable form: can we just use it both places?

Reply:
Why not use either `pbench_admin_token` or `pbench_token` as appropriate?

Reply:
Yes for the `pbench_admin_token` but no for the `pbench_token` because `pbench_token` only generate valid pbench token for the 'drb' user. Here we are trying to generate tokens for any user specified by the `user` fixture.

Reply:
Ah....  🤔